README.md

```diff
@@ -62,6 +62,9 @@ This file should contain the variables:
 `database_username`
 `database_password`
 `database_name`
+`aws_secret_access_key`
```

Comment:
Very detailed README.md, good work! Need to take another look at mine and add to it based on yours!